commands/inspect.go

```diff
@@ -0,0 +1,240 @@
+	i.printf("%-18s %s\n", "Key ID:", cert.KeyId)
+	i.printf("%-18s %v\n", "Principals:", cert.ValidPrincipals)
+	i.printf("%-18s %s\n", "Valid After:", formatTime(cert.ValidAfter))
+	i.printf("%-18s %s\n", "Valid Before:", formatTime(cert.ValidBefore))
+	i.printf("%-18s %v\n", "Critical Options:", cert.CriticalOptions)
+
+	// Format extensions nicely
+	i.printf("Extensions:\n")
+	for key, value := range cert.Extensions {
+		if key == "openpubkey-pkt" {
+			i.printf("  %s: [PKToken data]\n", key)
+		} else {
+			i.printf("  %s: %s\n", key, value)
+		}
+	}
+
+	// Extract openpubkey-pkt extension if it exists
+	pktStr, ok := cert.Extensions["openpubkey-pkt"]
+	if !ok {
+		i.printf("\nNo openpubkey-pkt extension found\n")
+		return
+	}
+
+	i.inspectPKToken(pktStr)
+}
+
+// formatTime converts a Unix timestamp to a readable date string
+func formatTime(timestamp uint64) string {
+	if timestamp == 0 {
+		return "Not set"
+	}
+	if timestamp == 1<<64-1 {
+		return "Forever"
+	}
+	t := time.Unix(int64(timestamp), 0)
+	return t.Format(time.RFC3339)
+}
+
+func (i *InspectCmd) inspectPublicKey(pubKey ssh.PublicKey) {
+	i.printf("--- SSH Public Key Information ---\n")
+	i.printf("Type: %s\n", pubKey.Type())
+
+	// Get fingerprint
+	fingerprint := ssh.FingerprintSHA256(pubKey)
+	i.printf("Fingerprint: %s\n", fingerprint)
+
+	// Get marshal format
+	marshal := base64.StdEncoding.EncodeToString(pubKey.Marshal())
+	i.printf("Marshal (base64): %s...\n", marshal[:20])
+}
+
+func certificateType(certType uint32) string {
+	switch certType {
+	case ssh.UserCert:
+		return "User Certificate"
+	case ssh.HostCert:
+		return "Host Certificate"
+	}
+	return fmt.Sprintf("Unknown (%d)", certType)
+}
+
+func (i *InspectCmd) inspectPKToken(pktStr string) {
+	// Parse the PKToken
+	pkt, err := pktoken.NewFromCompact([]byte(pktStr))
+	if err != nil {
+		i.printf("Error parsing PKToken: %v\n", err)
+		return
+	}
+
+	// Print token structure and metadata
+	i.printf("\n--- PKToken Structure ---\n")
+	i.printf("Payload:\n")
+	i.printJSON(pkt.Payload)
+
+	// Print signature information
+	i.printf("\n--- Signature Information ---\n")
+	if pkt.Op != nil {
+		i.printf("Provider Signature (OP) exists\n")
+	}
+	if pkt.Cic != nil {
+		i.printf("Client Signature (CIC) exists\n")
+	}
+	if pkt.Cos != nil {
+		i.printf("Cosigner Signature (COS) exists\n")
+	}
+
+	// Print token metadata
+	i.printf("\n--- Token Metadata ---\n")
+	i.printTokenMetadata(pkt)
+
+	// Extract ID Token information
+	i.printf("\n--- ID Token Payload ---\n")
+	var idTokenInfo struct {
+		OpToken string `json:"op_token"`
+	}
+	if err := json.Unmarshal(pkt.Payload, &idTokenInfo); err != nil {
+		i.printf("Error extracting ID token: %v\n", err)
+	} else {
+		parts := strings.Split(idTokenInfo.OpToken, ".")
+		if len(parts) >= 2 {
+			payload, err := base64URLDecode(parts[1])
+			if err != nil {
+				i.printf("Error decoding ID token payload: %v\n", err)
+			} else {
+				i.printJSON(payload)
+			}
+		}
+	}
```

Comment:
This doesn't print anything out because pkt.Payload is already the ID Token JSON payload. There is no op_token to be found inside.

You can delete this whole block because you already printed out the ID Token payload above when you printed out

```
--- PKToken Structure ---
Payload:
{
...
```

I suspect this was earlier code that accidently got left in. I missed it on my first few read throughs.